sdk/communication/azure-communication-administration/samples/phone_number_orders_sample.py

```diff
@@ -73,16 +73,13 @@ def get_reservation_by_id():
 
 def begin_reserve_phone_numbers():
     # [START begin_reserve_phone_numbers]
-    reservationOptions = CreateSearchOptions(
+    reserve_phone_numbers_response = phone_number_administration_client.begin_reserve_phone_numbers(
```

Comment:
It is a poller rather than a response, right?

Reply:
True. Corrected.